src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.ts

```diff
@@ -97,4 +102,60 @@ export class EditKeyboardMappingComponent implements OnInit {
   public cancel(): void {
     this.onCancel.emit()
   }
+
+  public get formValidationTooltip(): string | undefined {
+    const invalidControls = this.findInvalidControls()
+    return invalidControls.length > 0 ? $localize`common.form-required-field.button-tooltip` + `: ${invalidControls.join(', ')}` : undefined
+  }
+
+  private findInvalidControls(): string[] {
+    const invalidControlLabels = []
+    const controls = { ...this.actionTriggerForm.controls, ...this.actionTriggerDataForm.controls }
+    for (const name in controls) {
+      if (controls[name].valid) {
+        continue
+      }
+      const translateControl = this.translateControl(name)
+      if (translateControl) {
+        invalidControlLabels.push(translateControl)
+      }
+    }
+    return invalidControlLabels
+  }
```

Comment:
Same comments in here as for EditShelfActionPanelConfigurationComponent.

Reply:
Fixed